release.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+function usage() {
+    echo
+    echo "Changes the image.*.yaml files and regenerates the build directories"
+    echo 'Commits the changes on a branch release_$RELEASE, ready for a push'
+    echo
+    echo "Usage: release.sh [options]"
+    echo
+    echo "required arguments"
+    echo
+    echo "  -r RELEASE          The oshinko release number, like v0.4.4"
+    echo
+    echo "optional arguments:"
```

Comment:
We specify the default for OSHINKO_VERSION, but the others do not list their default values.  Should probably include a note about what happens if they are not specified.